mobile/i18n.js

```diff
@@ -21,10 +21,7 @@ const LOCALES = REACT_APP_LOCALES;
 let FALLBACKTRANSLATIONS = {};
 
 try {
-  const pth = `../../../../${REACT_APP_PUBLIC_LOCALES_OUTPUT}`;
-  FALLBACKTRANSLATIONS = require(pth);
-  // console.debug('~~~ FALLBACK TRANSLATIONS ~~~');
-  // console.debug(FALLBACKTRANSLATIONS);
+  FALLBACKTRANSLATIONS = require("~/" + REACT_APP_PUBLIC_LOCALES_OUTPUT + "/index.js");
```

Comment:
Looks good 👍 

If you want, you can use [template strings](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/template_strings) to concat strings like this more easily